content/guides/references/release-stages.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Release Stages
+---
+
+Cypress follows Semantic Versioning so that it is clear how you may be affected
+when upgrading to a newer version of Cypress. If there is a minor version
+increment, 11.0.0 → 11.1.0 for example, you should not experience disruption in
+your usage of Cypress. If there is a major version increment, 11.1.0 → 12.0.0
+for example, you may need to make changes to continue using Cypress. For most
+scenarios this is a valid way to consider complexity when upgrading Cypress. In
```

Comment:
complexity or disruptive changes. personally dont love the term complexity because not all major version bumps are complex, just something we've changed will def require action from the user and/or break their current usage.

Reply:
How about this?
```suggestion
scenarios this is a valid way to consider how disruptive upgrading to the next version of Cypress may be. In
```